ctapipe/utils/datasets.py

```diff
@@ -25,14 +25,28 @@
 
 logger = logging.getLogger(__name__)
 
-__all__ = ["get_dataset_path", "find_in_path", "find_all_matching_datasets"]
+__all__ = [
+    "get_dataset_path", "find_in_path", "find_all_matching_datasets",
+    "get_default_url",
+]
 
 
 DEFAULT_URL = "http://cccta-dataserver.in2p3.fr/data/ctapipe-test-data/v1.1.0/"
 
 
-def get_searchpath_dirs(searchpath=None, url=DEFAULT_URL):
+def get_default_url():
```

Comment:
I dont think this should be called `get_default_url` if it does more than returning `DEFAULT_URL`.
`get_dataset_url` to be consistent with `get_dataset_path`?

Reply:
It returns the default for the functions.

E.g. all the functions have `url=None` and if `url is None` they do `get_default_url`. So it is indeed the default url.

Reply:
Then maybe remove the variable and put it into the function as its not used outside anymore?